lib/matplotlib/axis.py

```diff
@@ -1326,7 +1328,8 @@ def get_minorticklocs(self):
         tol = (hi - lo) * 1e-5
         minor_locs = [
             loc for loc, tr_loc in zip(minor_locs, tr_minor_locs)
-            if not np.isclose(tr_loc, tr_major_locs, atol=tol, rtol=0).any()]
+            if (not remove_overlaps or
```

Comment:
I think the check for remove_overlaps should go outside the list comprehension so it doesn't get called if there is no removal going on?

Reply:
sigh, that causes pep8 frustrations (the obvious way is 1 character too long!).